### ISSUE TITLE
Option to select file complement as a plain text.

### ISSUE THREAD

**Issue by diiiv9** [tadashi-aikawa/obsidian-various-complements-plugin#240]
Since I rely heavily on the plugin for autocompletion , It would be nice to have an option that selects a file complement and places it as a plain text (instead of a link) , this would help avoid mentioning the link repeatedly in the document. 

 Maybe having separate keys (eg : Enter and Tab ) to enter the complement as a link and as a plain text could help with this.
**_or_** maybe showing a plain text option if the link is already used in the file.

Grateful for the plugin !
Thanks !

**Comment by tadashi-aikawa** [comment]
Hi, @diiiv9 :)

Maybe, The two-step completion fulfill your requirements. If not, I'll consider implementing what you wrote.

https://github.com/tadashi-aikawa/obsidian-various-complements-plugin/issues/156

**Comment by diiiv9** [comment]
Sorry I don't understand what you mean by _two-step completion_ , I looked through the settings and couldn't find it.

**Comment by tadashi-aikawa** [comment]
@diiiv9 
Please show #156. That feature is experimental now.

**Comment by diiiv9** [comment]
I'm sorry that doesn't help my use case. 
Say I'm writing an article on the topic of "politics". The article would use the word _politics_ frequently and the VC plugin would suggest the Internal Link Complement on a note named "Politics" every time. However after I have linked to the note the first time I would either 
-like that the _non link_ version of the word be suggested as a higher priority.
- or I have the ability to press a key and the word entered is a _non link_  version.

This is because I don't want create a link every time I mention the word.
FWIW the plugin does show a _non link_ version of the word after I have typed it out the first time , however it is ranked lower than the internal link version. 

A ranking change could fix the problem. If VC detects that there is a non link version of the same word in the current file it should be ranked above the link. Basically an toggle option like _"Lower suggestion priority for multiple links to same note"_.

Thanks !
<img width="513" alt="Screenshot 2023-08-19 at 4 17 50 PM" src="https://github.com/tadashi-aikawa/obsidian-various-complements-plugin/assets/11439193/fed1a057-0dfc-4b05-973d-bd9bd3005157">

**Comment by aubreyz** [comment]
Surely it shows the most frequently used on the top. I can't see any reason to complicate that (you could imagine all sorts of other possible toggles - prefer capitalised words, prefer longer, prefer shorter, prefer words with special characters etc etc).

**Comment by tadashi-aikawa** [comment]
@diiiv9 
OK. I think that your FR, "Option to select file complement as a plain tex" is the best solution so far.

**Comment by diiiv9** [comment]
@tadashi-aikawa Thank you for your time 🙏

**Comment by tadashi-aikawa** [comment]
After #245

**Comment by tadashi-aikawa** [comment]
@diiiv9 
I have released it in [v9.0.0-beat2](https://github.com/tadashi-aikawa/obsidian-various-complements-plugin/releases/tag/9.0.0-beta2) 🚀 

If you use BRAT, I would appreciate it if you could check it out before the official release.

The default hotkey is nothing, and you need to set a hotkey, like `Tab`.

![Image](https://github.com/tadashi-aikawa/obsidian-various-complements-plugin/assets/9500018/572f0d7b-d73c-4c4a-9eec-12fc06ec5727)

**Comment by tadashi-aikawa** [comment]
@diiiv9 
There are a few bugs in beta2, so I have fixed it and released it in beta3.

**Comment by diiiv9** [comment]
This perfect ! 
A minor suggestion would be to name the option as "Insert _link_ as text" maybe.

Thanks !!

**Comment by tadashi-aikawa** [comment]
@diiiv9 
Thank you for your confirmation and suggestion👍 

In actual, the command acts as "Insert anything as text," so I keep the name "insert as text".
